### PR TITLE
kubeedge: fixed golint warnings

### LINKED ISSUE #25: fix golint warnings
Currently code in many directories (excluding `vendor`, `docs`, etc.) fails to pass the `golint`.

Contributors are very welcome to take part in this work. Just leave the directory you want to fix in this issue.

How to do it?  
A simple way is 
```shell
cd ${GOPATH}/src/github.com/kubeedge/kubeedge
# get gometalinter(https://github.com/alecthomas/gometalinter)
curl -L https://git.io/vp6lP | sh
export PATH=${PATH}:${GOPATH}/src/github.com/kubeedge/kubeedge/bin
gometalinter --disable-all --enable=golint --include=<Directory-You-Want-To-Fix> ./...
```
Then you'll see warnings if any. Just fix them accordingly.

/cc @m1093782566 @Lion-Wei @stewart-yu

### PR DESCRIPTION
**Pull request by maurya-anuj**
Fixed golint warnings in beehive, cmd, pkg folder

Fixes #25

### CODE REVIEW

**Comment by sids-b** [comment]
Thanks a lot @maurya-anuj . These contributions help very much in improving quality and code understanding. 

Will review soon

File: pkg/common/dbm/db.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/kubeedge/kubeedge/beehive/pkg/common/log"
 
 	"github.com/astaxie/beego/orm"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/mattn/go-sqlite3" //Blank import to run only the _inti function
```
Comment (sids-b) [inline pkg/common/dbm/db.go]: nit: _inti -> init

File: beehive/pkg/core/model/event.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package model
 
+//constants for cloud connection
 const (
-	CLOUD_CONNECTED    = "cloud_connected"
-	CLOUD_DISCONNECTED = "cloud_disconnected"
+	CloudConnect    = "cloud_connected"
```
Comment (sids-b) [inline beehive/pkg/core/model/event.go]: rename to CloudConnected ? and CloudDisconnected

File: pkg/edged/metaclient/configmap.go [hunk 3]
```diff
@@ -65,9 +68,10 @@ func (c *configMaps) Get(name string) (*api.ConfigMap, error) {
 
 	if msg.GetOperation() == model.ResponseOperation {
 		return handleConfigMapFromMetaDB(content)
-	} else {
-		return handleConfigMapFromMetaManager(content)
 	}
+	//else
```
Comment (sids-b) [inline pkg/edged/metaclient/configmap.go]: no need to comment else..... applicable to all similar places

File: pkg/edged/metaclient/metaclient.go [hunk 4]
```diff
@@ -83,14 +86,15 @@ func (s *send) SendSync(message *model.Message) (*model.Message, error) {
...
 			return true, nil
-		} else {
-			if retries < 3 {
-				log.LOGGER.Errorf("send sync message %s failed, error:%v, retries: %d", message.GetResource(), err, retries)
-				return false, nil
-			} else {
-				return true, err
-			}
 		}
+		//else
```
Comment (sids-b) [inline pkg/edged/metaclient/metaclient.go]: remove else

File: pkg/edged/util/util.go [hunk 5]
```diff
@@ -199,9 +205,10 @@ func GetMatchingGlobalIP(addrs []net.Addr, family AddressFamily) (net.IP, error)
 				if ip.IsGlobalUnicast() {
 					log.LOGGER.Infof("IP found %v", ip)
 					return ip, nil
-				} else {
-					log.LOGGER.Infof("Non-global unicast address found %v", ip)
 				}
+				//else
```
Comment (sids-b) [inline pkg/edged/util/util.go]: remove comment

File: pkg/edged/util/util.go [hunk 6]
```diff
@@ -199,9 +205,10 @@ func GetMatchingGlobalIP(addrs []net.Addr, family AddressFamily) (net.IP, error)
...
+				log.LOGGER.Infof("Non-global unicast address found %v", ip)
+
```
Comment (sids-b) [inline pkg/edged/util/util.go]: remove blank line

File: pkg/eventbus/mqtt/server.go [hunk 7]
```diff
@@ -31,8 +31,8 @@ import (
 	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
 )
 
-// A MqttServer serve as an internal mqtt broker.
-type MqttServer struct {
+// A Server serve as an internal mqtt broker.
```
Comment (sids-b) [inline pkg/eventbus/mqtt/server.go]: A server -> Server ?

File: pkg/metamanager/msg_processor.go [hunk 8]
```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/kubeedge/kubeedge/pkg/metamanager/dao"
 )
 
+//constants for
```
Comment (sids-b) [inline pkg/metamanager/msg_processor.go]: constants for ??

File: vendor/k8s.io/client-go/util/flowcontrol/backoff.go [hunk 9]
```diff
@@ -29,7 +29,7 @@ type backoffEntry struct {
 	lastUpdate time.Time
 }
 
-type Backoff struct {
+type  Backoff struct {
```
Comment (sids-b) [inline vendor/k8s.io/client-go/util/flowcontrol/backoff.go]: Dont make changes in vendor file...

File: beehive/pkg/core/context/context.go [hunk 10]
```diff
@@ -24,7 +26,7 @@ type MessageContext interface {
 	Send2GroupSync(moduleType string, message model.Message, timeout time.Duration) error
 }
 
-// global context
+// Context : global context object
```
Comment (sids-b) [inline beehive/pkg/core/context/context.go]: Context : -> Context is

File: beehive/pkg/core/core.go [hunk 11]
```diff
@@ -24,7 +24,7 @@ func StartModules() {
 	}
 }
 
-// if get the special signals, cleanup modules
+// GracefulShutdown : if get the special signals, cleanup modules
```
Comment (sids-b) [inline beehive/pkg/core/core.go]: GracefulShutdown : -> GracefulShutdown is ?
Applicable to to all similar places

**Comment by m1093782566** [comment]
Thanks for diligent review! @sids-b 

We can merge this PR now, I think :)

**Pull request merged**